core/src/labels/label.cpp

```diff
@@ -264,31 +260,40 @@ bool Label::updateState(const glm::mat4& _mvp, const glm::vec2& _screenSize, flo
         enterState(State::out_of_screen, 0.0);
     }
 
+    return true;
+}
+
+bool Label::evalState(const glm::vec2& _screenSize, float _dt) {
```

Comment:
Probably make sense to put asset statements here for purely illegal state transitions..

example, a label should not go from fading_in to wait_occ or dead!